src/components/Card/Card.js

```diff
@@ -1,31 +1,54 @@
-import React from 'react';
-import "./Card.css"
-const Card = ({id,title,description,date,name,age}) => {
-    return (
-        <div className="post__container">
-            <div className="post">
-                <div className="post-content">
-                    <strong>{id}</strong>
-                    <strong>{title}</strong>
-                    <div>
-                        {description}
-                    </div>
-                    <div>
-                        {date}
-                    </div>
-                    <div>
-                        {name}
-                    </div>
-                    <div>
-                        {age}
-                    </div>
-                </div>
-                <div className="post__buttons">
-                    <button>Удалить</button>
-                </div>
+import React from "react";
+import "./Card.css";
+import { buildDescription } from "./summaryBuilder";
+import { usdTo } from "../../utils/moneyConverter";
+import { ICONS } from "../../constants/icons";
+import { CURRENCY } from "../../constants/currency";
+
+const Card = ({
+  mark,
+  model,
+  generation,
+  year,
+  transmission,
+  engine,
+  mileage,
+  image,
+  price,
+}) => {
+  const description = buildDescription({ year, transmission, engine, mileage });
+  const secondaryCurrency = CURRENCY.EUR;
+
+  return (
+    <div className="card__container">
+      <div className="card__image">
+        <img src={image} alt="#" />
```

Comment:
Please set some description to alt attribute 
It is not a good idea to live it blank or use "#"